scalyr_agent/platform_posix.py

```diff
@@ -720,8 +720,8 @@ def stop_agent_service(self, quiet):
 
         # Try killing the daemon process
         try:
-            # Do 5 seconds worth of TERM signals.  If that doesn't work, KILL it.
-            term_attempts = 50
+            # Do 50 seconds worth of TERM signals.  If that doesn't work, KILL it.
+            term_attempts = 500
```

Comment:
Could this increase in shutdown delay affect other setups / configurations when running syslog monitor? Perhaps Kubernetes one where we have health checks and shut down grace period defined?

Reply:
That's a good point. Also, this change shouldn't be part of this PR. Reverting.